pkg/server/proto/server.proto

```diff
@@ -902,6 +902,22 @@ message Ref {
   message Trigger {
     string id = 1;
   }
+
+  // Job references a Job message value by job id.
+  message Job {
+    string id = 1;
+  }
+
+  // TrackTask references a TrackTask message by its id or the job id it queued
+  message TrackTask {
+    oneof ref {
+      // the id of the tracktask record
+      string id = 1;
+
+      // The main job ID that the task initiated
+      string job_id = 2;
```

Comment:
Neat! There are 3 tasks revolving around a task: start, run, stop. Its unclear to me as a first time reader what this is referencing, it sounds like "run". Could you clarify in doc comments?

Reply:
Oh that's right! It's the run job. Will clarify in the comments 👍🏻